### DISCUSSION TITLE
Add at calculated column for days to date. Its possible in dataview but not Projects?

### DISCUSSION THREAD

**Discussion by cwfmoore** [marcusolsson/obsidian-projects#702] [Help]
In dataview I am able to calculate the number of days until a date in frontmatter using:

```dataview
TABLE status, deadline, deadline - date(today) AS "Remaining Time"
WHERE contains(tags, "project")
```

Entering the same query into a project does not however give me the "Remaining Time" column. Is this possible another way?

**Comment by Acylation** [comment]
Unfortunately the derived / calculated fields are not supported yet.
